docs/content/guides/developer/app-examples/coin-flip.mdx

```diff
@@ -433,208 +453,297 @@ Now take a look at the main function in this game, `finish_game`:
-
-```
-pnpm create @mysten/dapp
-```
+:::
 
-This CLI command prompts you through a couple of steps:
+### Overview
 
-1. It asks you the starter template that you want to use. Currently, there are two variants:
+The user interface (UI) of this frontend example demonstrates how to use the dApp Kit instead of serving as a production-grade product, so the Player and the House features are in the same UI to simplify the process. In a production solution, your frontend would only contain functionality dedicated to the Player, with a backend service carrying out the interactions with House functions in the smart contracts.
 
-   1. `react-client-dapp`: This starter template contains the minimum dApp Kit template code that you can start with. This variant is meant for developers already familiar with the dApp Kit and who don't want unnecessary template code.
-   1. `react-e2e-counter`: This starter template contains a simple counter Sui Move smart contract with the frontend template code interacting with it. This variant is meant for developers trying to learn how to use dApp Kit.
+The UI has two columns:
 
-1. It prompts you to name your project folder.
+- First column is dedicated to the Player, and all Player-related features live there
+- Second column is dedicated to the House, and all House-related features live there
 
-Done. Your project has all necessary code to get you started. Lastly, `cd` into your project folder and run `pnpm install` to install all dependencies.
+### Scaffold a new app
 
-### User interface layout design
+The first step is to set up the client app. Run the following command to scaffold a new app.
 
-The user interface (UI) of this frontend example demonstrates how to use the dApp Kit instead of serving as a production-grade product, so the Player and the House features are in the same UI to simplify the process. In a production solution, your frontend would only contain functionality dedicated to the Player, with a backend service carrying out the interactions with House functions in the smart contracts.
+```bash
+pnpm create @mysten/dapp --template react-client-dapp
+```
 
-The UI has two columns:
+or
 
-- First column is dedicated to the Player, and all Player-related features live there
-- Second column is dedicated to the House, and all House-related features live there
+```bash
+yarn create @mysten/dapp --template react-client-dapp
+```
 
 ### Project folder structure
 
 Structure the project folder according to the UI layout, meaning that all Player-related React components reside in the `containers/Player` folder, while all House-related React components reside in the `containers/House` folder.
 
+### Connecting your deployed package {#connecting-your-package}
+
+Add the `packageId` value you saved from [deploying your package](#deployment) to a new `src/constants.ts` file in your project:
```

Comment:
Just a note that past tense is good here (saved) because you're referencing a previous step. So maybe "pretty much always use present tense" is a bit strong....